boards/sparky_v2.cpp

```diff
@@ -115,9 +115,9 @@ bool Sparky_v2::init(void)
     // -------------------------------------------------------------------------
     // Init LEDs
     // -------------------------------------------------------------------------
-    ret &= led_err_gpio_.init();
-    ret &= led_stat_gpio_.init();
-    ret &= led_rf_gpio_.init();
+    ret = led_err_gpio_.init();
+    ret = led_stat_gpio_.init();
+    ret = led_rf_gpio_.init();
```

Comment:
Wouldn't this erase the value of ret at each led initialization? For example, if the led_err_gpio init fails but the two others succeed then ret is true.